docs/en/drivers/android-selendroid.md

```diff
@@ -0,0 +1,52 @@
+## The Selendroid Driver for Android
+
+> **Note**: This driver is _DEPRECATED_ and should not be used unless
+> absolutely necessary. The information in this doc may not keep up to date
+> with reality, and the driver will be removed in a future version of Appium.
+> To begin Android automation with Appium today, please use the [UiAutomator2
+> Driver](/docs/en/drivers/android-uiautomator2.md) instead.
+
+Appium's support for very old versions of Android (4.1 and below) comes via
+a project called [Selendroid](http://selendroid.io). Selendroid is an
+Instrumentation-based automation framework.
+
+Development of the Selendroid driver happens at the
+[appium-selendroid-driver](https://github.com/appium/appium-selendroid-driver)
+repo.
+
+### Requirements and Support
+
+In addition to Appium's general requirements:
+
+* Java 6+ installed and configured correctly for your platform
+* Mac, Windows, or Linux OS with the ability to run the Android SDK
+
+### Usage
+
+The way to start a session using the Espresso driver is to include the
+`automationName` [capability](#TODO) in your [new session request](#TODO), with
+the value `Selendroid`. Of course, you must also include appropriate
+`platformName` (=`Android`), `platformVersion`, `deviceName`, and `app`
+capabilities, at a minimum.
+
```

Comment:
This looks like a cut-and-paste from the Espresso driver doc

Reply:
it is the same information, but having it in both places is important.

Reply:
See my new comment.